view/frontend/templates/checkout/cart.phtml

```diff
@@ -1,10 +1,11 @@
 <?php if ( $this->isKlaviyoEnabled() && $this->getPublicApiKey() ): ?>
     <script>
         require([
-           'jquery'
-        ], function (jQuery) {
+           'jquery',
+           'mage/url'
+        ], function (jQuery, url) {
           jQuery.ajax({
-              url: window.location.protocol + '//' + window.location.hostname + '/reclaim/checkout/reload',
+              url: url.build('/reclaim/checkout/reload'),
```

Comment:
does url.build not add the `/` ? it may handle it for both cases just want to make sure since we use it with and w/out a `/` as a prefix on the url

Reply:
Tested this out - seems url.build is smart enough to put the slash in between the base URL and the provided route if it's not there already